lib/soap/request.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Soap.Request do
+  defstruct [:method, :url, :body, :headers, :options]
+  @moduledoc """
+  Documentation for Soap.Request.
```

Comment:
Let's write the documentation for models and methods right away

Reply:
Done.